mono/mini/method-to-ir.c

```diff
@@ -92,6 +92,17 @@ static const gboolean debug_tailcall_break_run = FALSE;     // insert breakpoint
 static const gboolean debug_tailcall_try_all = FALSE;       // consider any call followed by ret
 static const gboolean debug_tailcall = FALSE;               // logging
 
+static void
+tailcall_print (const char *format, ...)
```

Comment:
This has an affect of adding a few more strings to the image than before, because the optimizer was previously presented with a static const false and now an environment variable. Ok?